docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -18,3 +18,23 @@ import AuthZ from 'keycloak-js/authz';
 The `spi-truststore-file-*` options and the truststore related options `https-trust-store-*` are deprecated, please use the new default location for truststore material, `conf/truststores`, or specify your desired paths via the `truststore-paths` option. For details refer to the relevant https://www.keycloak.org/server/keycloak-truststore[guide].
 
 The `tls-hostname-verifier` property should be used instead of the `spi-truststore-file-hostname-verification-policy` property.
+
+= Deprecated `--proxy` option
+
+The `--proxy` option has been deprecated and will be removed in a future release. The following table explains how the deprecated option maps to supported options.
+
+[cols="1,1"]
+|===
+|`kc.sh` (no `proxy` option set)
+|`kc.sh`
```

Comment:
Is it necessary to have it here? Seems irrelevant to me in the context of `Deprecated --proxy option`. Only users who have specified the option are affected by this deprecation. WDYT?

Reply:
I wanted to make the comparison as explicit as possible, incl. how no proxy maps to no proxy headers. Does it make any sense? :)